CHANGES.rst

```diff
@@ -10,6 +10,20 @@ New Features
 
 - Exact-text filtering for metadata plugin. [#2147]
 
+- CLI launchers no longer require data to be specified. [#1890]
+
+Mosviz
```

Comment:
im assuming you just need to rebase which is why this extra stuff is in here

Reply:
Yikes, I think this resulted from a bad rebase. I'll fix that now. Good catch!